crowbar_framework/app/models/deployer_service.rb

```diff
@@ -181,6 +162,27 @@ def transition(inst, name, state)
       roles << node.run_list_to_roles
       roles.flatten!
 
+      # Allocate required addresses
```

Comment:
Should we check if an address has already been allocated and not do anything if that's the case?